int16 low_h
int16 low_s
int16 low_v
int16 high_h
int16 high_s
int16 high_v
int16 dilation_element
int16 dilation_size